infra/src/main/resources/application.yml

```diff
@@ -23,6 +23,7 @@ spring:
       connection-timeout: 10000
       leak-detection-threshold: 60000
       validation-timeout: 5000
+      connection-init-sql: "set jit=off;"
```

Comment:
Tähän voisi mielestäni kommentoida että minkä takia JIT on asetettu pois päältä. On mahdollista että joskus hamassa tulevaisuudessa päädytään koodaamaan jotain supermassahakuja ja ihmetellään miksi niissä kestää vaikka ei kuuluisi, ja silloin a) tämän `set jit=off`:in olemassaolon huomaaminen helpottuu kun se ei ole vaan rivi konffimassan joukossa ja b) saadaan palautettua helpommin mieleen miksi tuohon on ylipäätään päädytty